class.two-factor-core.php

```diff
@@ -122,10 +122,10 @@ function show_two_factor_login( $user ) {
 
 		$login_nonce = $this->create_login_nonce( $user->ID );
 		if ( ! $login_nonce ) {
-			wp_die( __( 'Could not save login nonce.', 'two-factor' ) );
+			wp_die( esc_html__( 'Could not save login nonce.', 'two-factor' ) );
 		}
 
-		$redirect_to = isset( $_REQUEST['redirect_to'] ) ? $_REQUEST['redirect_to'] : $_SERVER['REQUEST_URI'];
+		$redirect_to = isset( $_POST['redirect_to'] ) ? $_POST['redirect_to'] : $_SERVER['REQUEST_URI'];
```

Comment:
Leave this as $_REQUEST -- it's necessary if switching to a different provider on a branch I'm working on.